python/simplesauce/session.py

```diff
@@ -81,5 +81,20 @@ def start(self):
         )
         return self.driver
 
-    def stop(self):
+    def stop(self, result=None):
+        self.update_test_result(result)
         self.driver.quit()
+
+    def update_test_result(self, result_in):
+        result = ''        
+
+        if result_in == True:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to True should be 'if cond is True:' or 'if cond:'](https://app.codacy.com/app/SauceLabs/simple_sauce/pullRequest?prid=4626732)